### DISCUSSION TITLE
GitHub has a problem with the ping

### DISCUSSION THREAD

**Discussion by ismilearefin** [community/community#42644] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

![git](https://user-images.githubusercontent.com/92626163/209516350-28f70f78-57fd-4f16-bd00-d70284c9d597.JPG)
Ping statistics for my computer is not working, 100% loss

**Comment by jfullstackdev** [comment]
it's really not working

![image](https://user-images.githubusercontent.com/47092464/209560933-7fe4be13-9951-4171-8926-35f2df55dbfb.png)

I guess that GitHub intentionally does not allow that, some sites are doing that
